cmake/patches/vc-deprecated-and-bit-scan-forward.patch

```diff
@@ -0,0 +1,58 @@
+commit d19a82411eb2f478671f20f4a3ed80d4820cf672
+Author: Guilherme Amadio <amadio@cern.ch>
+Date:   Fri Mar 6 10:46:02 2020 +0100
+
+    Avoid redefinition of _bit_scan_{forward,reverse} macros
```

Comment:
Can you update the commit log? Actually - given that this is a git patch with a sha you cannot just modify the diff :-) Could you instead add another patch? @amadio what do you suggest here?

Reply:
I just `cherry-pick` commit from @amadio repository and produce cumulative patch from two commits. I have no idea how many patches can be applied using cmake `ExternalProject`

Reply:
I created the diff with `git format-patch`, that's why there's a hash there. For ROOT, that hash is meaningless. @linev added another commit (with its own commit message) below, which is fine. However, I think that the patch should only get rid of the warning by removing the line I comment below rather than comment that whole template.